### PR TITLE
Torrent list design

### PR DESCRIPTION
**Pull request by dcposch**
## Removed hover / select brightness changes

All those gradients and brightness filters toggling on and off were getting annoying

### Before

before

### After

after

### CODE REVIEW

**Comment by dcposch** [comment]
### Progress bar styling

This fixes the inconsistency between different platforms.

screen shot 2016-09-02 at 12 25 00 pm

**Comment by Flet** [comment]
Nice work!

**Comment by Goldob** [comment]
LGTM

File: src/renderer/pages/torrent-list-page.js [hunk 1]
```diff
@@ -46,11 +46,9 @@ module.exports = class TorrentList extends React.Component {
     // Background image: show some nice visuals, like a frame from the movie, if possible
     const style = {}
     if (torrentSummary.posterFileName) {
-      const gradient = isSelected
-        ? 'linear-gradient(to bottom, rgba(0, 0, 0, 0.8) 0%, rgba(0, 0, 0, 0.4) 100%)'
-        : 'linear-gradient(to bottom, rgba(0, 0, 0, 0.5) 0%, rgba(0, 0, 0, 0) 100%)'
+      const gradient = 'linear-gradient(to bottom, rgba(0, 0, 0, 0.5) 0%, rgba(0, 0, 0, 0) 100%)'
       const posterPath = TorrentSummary.getPosterPath(torrentSummary)
-      style.backgroundImage = gradient + `, url('${posterPath}')`
+      style.backgroundImage = `${gradient}, url('${posterPath}')`
```
Comment (demoneaux) [inline src/renderer/pages/torrent-list-page.js]: Hi

**Pull request merged**